addons/website_slides/static/lib/pdfslidesviewer/PDFSlidesViewer.js

```diff
@@ -148,6 +148,23 @@ var PDFSlidesViewer = (function(){
     PDFSlidesViewer.prototype.toggleFullScreen = function(){
         var el = this.canvas;
 
+        var $PDFViewer = $("div#PDFSlideViewer")
+        var $navigation_footer = $('div#PDFViewer > div.navbar-fixed-bottom')
```

Comment:
It's a separate parent div containing footer, and so giving this div max z-index will only make all its inner content visible.